services/simplified_crontab

```diff
@@ -41,6 +41,10 @@ HOME=/var/www/circulation
 # identifiers, 25 at a time.
 # */10 * * * * root core/bin/run metadata_wrangler_coverage >> /var/log/cron.log 2>&1
 
+# If any works are classified under unprocessed subjects, reclassify
+# those works.
+30 22 * * * root core/bin/run work_classify_unchecked_subjects
```

Comment:
Can you push the output into cron.log? The script output goes into its own file, but any output between calling the command and running the script (like in run, for example) can only be seen in cron.log.